libs/blocks/marketo/marketo.js

```diff
@@ -19,6 +19,7 @@ import {
   loadLink,
   localizeLink,
   createTag,
+  getConfig,
```

Comment:
We will need to call `setConfig` in the unit tests and update the import mapped marketo-utils.js.